test/project/partials/formats/erb/.gitkeep

```diff
@@ -0,0 +1 @@
+# Resolver breaks when files and dir's have the same name
```

Comment:
It only breaks in the case where there is a directory with the same name (minus extensions). Please add a specific testcase that shows this behaviour.